test/helpers/gem_helpers.rb

```diff
@@ -17,7 +17,7 @@ def build_gem(name, version, summary = "Gemcutter", platform = "ruby", &block)
     build_gemspec(new_gemspec(name, version, summary, platform, &block))
   end
 
-  def new_gemspec(name, version, summary, platform, &block)
+  def new_gemspec(name, version, summary, platform, metadata = nil, &block)
```

Comment:
check the next added line:
```ruby
s.metadata = metadata if metadata
```
if it would be `{}` then it could reset any defaults ... although there aren't any